execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -59,6 +61,18 @@ struct P2pTestCase {
 }
 
 // TODO: use LocalAccount instead
+fn LocalAccountGenerator(){
+    
+    let mut account = LocalAccount::generate;
+    let txn = account.sign_with_transaction_builder(
+        txn_factory.payload(
+        encode_..._script_function(...),
+    )
+    );
+}
+
+
+
```

Comment:
what we'd like here is to replace AccountData with LocalAccount since they achieve similar functionality